mkdocs.yml

```
@@ -69,7 +70,7 @@ plugins:
     - search
     - branchcustomization:
         update_config:
-         - branch: /(?!^master$)/
+         - branch: /(?!^bep020$)/
```

```suggestion
         - branch: /(?!^master$)/
```
so rendering does include intended to be there watermark for a "branched out" version.  If I got the purpose of this setting right, it better not to be changed in PR so it doesn't get by mistake merged into the master branch and require later a fix for that.

---

Worth adding a comment or warning in the yml next to it?

---

When we worked on that month ago it was necessary to use mkdocs serve command (at least this is what I remember), in the new version it is no longer the case so I put back ` branch: /(?!^master$)/` in bep020 branch too.